### PR TITLE
Sign in with Google OAuth/CWL

### PR DESCRIPTION
**Pull request by d3li0n**
# Description

Adds Google OAuth authentication.
Removes sign up page

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This requires a run of `yarn install`

# How Has This Been Tested?

- [x] Added unit tests
- [ ] Tested by the team

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where needed 
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/app/components/Settings/AddStudents.tsx [hunk 1]
```diff
@@ -105,7 +105,6 @@ function RenderTable({
                   avatar={<Avatar src={item.photoURL} />}
                   title={item.name}
                 />
-                <div>{item.email}</div>
```
Comment (d3li0n) [inline packages/app/components/Settings/AddStudents.tsx:108]: As per discussion with Ramon

File: packages/app/components/Settings/ProfileSettings.tsx [hunk 2]
```diff
@@ -1,103 +1,162 @@
...
-    await API.profile.patch(pick(newProfile, ['email']))
-    mutate()
+    let newProfile = null
+
+    if (profile && profile.accountType === AccountType.LEGACY) {
+      newProfile = { ...profile, ...updateProfile }
+      newProfile.sid = parseInt(newProfile.sid, 10)
+      mutate(newProfile, false)
+      if (profile.email === updateProfile.email) {
+        console.log(pick(newProfile, ['firstName', 'lastName', 'sid']))
```
Comment (Ferdinand737) [inline packages/app/components/Settings/ProfileSettings.tsx:22]: Remove this?

File: packages/app/pages/login.tsx [hunk 3]
```diff
@@ -1,24 +1,74 @@
...
   const [accountActiveResponse, setAccountActiveResponse] = useState(true)
+  const [loginMenu, setLoginMenu] = useState(false)
+  const [organization, setOrganization] = useState(null)
+
+  const { data: organizations } = useSWR(`api/v1/organization`, async () =>
+    API.organizations.getOrganizations(),
+  )
+
+  const loginWithGoogle = async () => {
+    console.log(organization.id)
```
Comment (Ferdinand737) [inline packages/app/pages/login.tsx:51]: remove this?

File: packages/app/pages/login.tsx [hunk 4]
```diff
@@ -1,24 +1,74 @@
...
+      .then((res) => {
+        Router.push(res.redirectUri)
+      })
+      .catch((err) => {
+        console.log(err)
+      })
+  }
+
+  const loginWithInstitution = async () => {
+    window.location.href = `/api/v1/auth/shibboleth/${organization.id}`
```
Comment (Ferdinand737) [inline packages/app/pages/login.tsx:63]: Why not use router?

**Review by Ferdinand737 — APPROVED** [review]

**Comment by Ferdinand737** [comment]
Changing profile picture was not working for me

**Pull request merged**